perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_resource_attribute_def_virt_googleGroupName.java

```diff
@@ -122,6 +124,11 @@ public boolean removeAttributeValue(PerunSessionImpl sess, Resource resource, Gr
 		return false;
 	}
 
+	@Override
+	public List<String> getDependencies() {
+		return Collections.singletonList(AttributesManager.NS_FACILITY_ATTR_DEF + ":googleGroupNameNamespace");
```

Comment:
Because the value of this attribute is not actually used. The attribute is obtained and the value is immediately changed.

Reply:
You are right. The best solution for now is to let it there commented, but also add information that we don't want to use this attribute in dependencies because it does not affect value of dependent attribute.